linkatos/confirmation.py

```diff
@@ -0,0 +1,18 @@
+import linkatos.printer as printer
+from .utils import is_fresh_url
+
+
+def update_confirmation_if_url(parsed_message, expecting_confirmation):
+    if is_fresh_url(expecting_confirmation, parsed_message['type']):
+        expecting_confirmation = True
+
+    return (expecting_confirmation)
+
+
+def process_confirmation_if_yn(parsed_message, expecting_confirmation):
```

Comment:
Is it because it's not obvious that inside the `if` the first returned is not clear tha it is `expecting_confirmation`?